### PR TITLE
Bug fix for _sortNewestFirst

### PR DESCRIPTION
**Pull request by josephui**
The comparator function `_sortNewestFirst` returns incorrect values, so that when used for sorting in `getLatest`, it result in incorrectly returned image.

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Sheesh, this was pretty wrong! Thanks!

File: test.js [hunk 1]
```diff
@@ -100,4 +100,33 @@ describe('gce-images', function () {
       });
     });
   });
-});
\ No newline at end of file
+
+  describe('_sortNewestFirst', function () {
+    describe('when imageA.creationTimestamp is newer than imageB.creationTimestamp', function () {
+      it('should return 1', function () {
```
Comment (josephui) [inline test.js]: But in `getLatest` (https://github.com/stephenplusplus/gce-images/blob/master/index.js#L131), `pop` is being used, so we want the "newest / latest" element as the last element of the sort.

Comment (josephui) [inline test.js]: Oh unless the intended behavior there is `shift`

Comment (stephenplusplus) [inline test.js]: Oops, can you change that to shift and then revert to the original behavior of the comparator?

Comment (josephui) [inline test.js]: Done, please verify.

**Comment by stephenplusplus** [comment]
Looks great, thank you for catching all of these bugs. Would you mind adding semicolons to match the style of the file? I'm happy to do this later as well.

**Comment by josephui** [comment]
Done. Would it be possible to bump the version on this and update https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/packages/compute/package.json#L58? It's hard to get around this bug in production code without hacks.

**Comment by stephenplusplus** [comment]
Yes, mobile now, but I will try to get a new Compute release out before the next couple of hours.

**Comment by josephui** [comment]
Thank you for all the quick feedback!

**Comment by stephenplusplus** [comment]
Finally got a connection. Working on this now :)

**Comment by stephenplusplus** [comment]
@josephui published under v0.3.1, so un- and re-installing `@google-cloud/compute` will pick up the fix. Thanks again for catching this *and* fixing it. Very much appreciated :)

**Pull request merged**